### PR TITLE
ECC-1102 Added Confirmation Content for Subscription with Eori number

### PR DESCRIPTION
**Pull request by vigneshwaripandi**

### CODE REVIEW

File: app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/SubscriptionDownloadController.scala [hunk 1]
```diff
@@ -42,7 +42,7 @@ class EoriDownloadController @Inject() (
     implicit request => _: LoggedInUserWithEnrolments =>
       cdsFrontendDataCache.sub02Outcome.map {
         case Sub02Outcome(processedDate, fullName, Some(eori)) =>
-          Right(eoriNumberDownloadView(eori, fullName, processedDate).body)
+          Right(subscriptionDownloadView(eori, fullName, processedDate).body)
```
Comment (NandanaRaghu) [inline app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/SubscriptionDownloadController.scala:45]: We can have a common controller for both standalone and combi journeys

File: app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/SubscriptionTextDownloadController.scala [hunk 2]
```diff
@@ -41,7 +41,7 @@ class EoriTextDownloadController @Inject() (
         name       <- cdsFrontendDataCache.sub02Outcome.map(_.fullName.trim)
         processedDate <- cdsFrontendDataCache.sub02Outcome
           .map(_.processedDate)
-      } yield Ok(eoriNumberTextDownloadView(eori, name, processedDate))
+      } yield Ok(subscriptionTextDownloadView(eori, name, processedDate))
```
Comment (NandanaRaghu) [inline app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/SubscriptionTextDownloadController.scala:44]: We can have a common controller for both standalone and combi journeys

File: app/uk/gov/hmrc/eoricommoncomponent/frontend/views/subscription_outcome.scala.html [hunk 3]
```diff
@@ -21,33 +21,40 @@
...
+<a class="govuk-link" href=@appConfig.checkEORINumber target="_blank" rel="noopener noreferrer">@messages("cds.subscription.outcomes.success.extra.information2")</a>
+}
 
 @layout_di(messages("cds.subscription.outcomes.success.title"), displayBackLink = false) {
 <div>
     @govukPanel(Panel(
-        title = HtmlContent(s"""${messages("cds.subscription.outcomes.success.heading", name)}<br><strong id="eori-number" class="govuk-!-font-weight-regular">$eori</strong>"""),
-        content = HtmlContent(s"""<p style="color:white" id="issued-date" class="govuk-heading-m">${messages("cds.subscription.outcomes.success.issued")} ${dateFormatter.format(processedDate)}</p>""")
+        title = HtmlContent(s"""${messages("cds.reg.existing.outcomes.success.heading.part1")} $name"""),
+        content = HtmlContent(s""" <h5 style="color:white" id="issued-date" class="govuk-heading-s"><strong>${messages("cds.subscription.outcomes.success.issued")} ${dateFormatter.format(processedDate)}</strong></h5><p style="color:white" id="eori-number" class="govuk-heading-m govuk-!-font-size-27">${messages("cds.subscription.outcomes.success.eori")} <br>$eori</p>""")
```
Comment (NandanaRaghu) [inline app/uk/gov/hmrc/eoricommoncomponent/frontend/views/subscription_outcome.scala.html:39]: do we need h5 here

**Review by NandanaRaghu — CHANGES_REQUESTED** [review]

File: app/uk/gov/hmrc/eoricommoncomponent/frontend/views/subscription_outcome.scala.html (hunk 3 above)
Comment (vigneshwaripandi) [reply app/uk/gov/hmrc/eoricommoncomponent/frontend/views/subscription_outcome.scala.html]: I have now changed the header order to h2 and small font

File: app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/SubscriptionDownloadController.scala (hunk 1 above)
Comment (vigneshwaripandi) [reply app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/SubscriptionDownloadController.scala]: I have removed the separate controller and now using the common controller for both standalone and combi journeys.

File: app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/SubscriptionTextDownloadController.scala (hunk 2 above)
Comment (vigneshwaripandi) [reply app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/SubscriptionTextDownloadController.scala]: I have removed the separate controller and now using the common controller for both standalone and combi journeys.

**Pull request merged**